src/sortStudents.ts

```diff
@@ -1,16 +1,79 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
-
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(students: Student[],
+  sortBy: SortType, order: SortOrder): Student[] {
+  let studentsCopy: Student[] = [];
+
+  studentsCopy = students.map((a) => ({ ...a }));
+
+  switch (sortBy) {
+    case SortType.Name:
+    case SortType.Surname:
+      if (order === 'asc') {
+        studentsCopy.sort((a: Student,
+          b: Student) => (a[sortBy].localeCompare(b[sortBy])));
```

Comment:
откуда такие переносы? просто обрыв строки
```suggestion
        studentsCopy.sort((a: Student, b: Student) => (
          a[sortBy].localeCompare(b[sortBy])
        ));
```